Source/Scene/ModelExperimental/buildDrawCommand.js

```diff
@@ -0,0 +1,49 @@
+import DrawCommand from "../../Renderer/DrawCommand.js";
```

Comment:
Oh @sanjeetsuhag I just checked the code coverage, looks like we need unit tests for this file and a couple other things:

![image](https://user-images.githubusercontent.com/8422414/128036885-10f8e04b-61bc-48b8-ae23-cd4e8ec913ad.png)

Reply:
I made a change to convert our pipeline stages into namespaces rather than functions. It's more typical, and it should give better test coverage.

Reply:
![image](https://user-images.githubusercontent.com/5172619/128418464-2acc8e86-8a69-4e7b-a5ca-f7f001f2f135.png)

I got the coverage up. There are couple of things that would be easily tested with a `verifyRender` call, but this is not ready for that yet since the fragment shader is so primitive.

Reply:
Yeah  that's fine. We should talk about when to add rendering tests soon (e.g. which branch to start introducing that)